components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -14936,6 +15191,41 @@ public void updateCategory(APICategory apiCategory) throws APIManagementExceptio
         return categoriesList;
     }
 
+    /**
+     * Get all available API categories of the organization
+     * @param organization
+     * @return
+     * @throws APIManagementException
+     */
+    public List<APICategory> getAllCategories(String organization) throws APIManagementException {
+
+        List<APICategory> categoriesList = new ArrayList<>();
+        try (Connection connection = APIMgtDBUtil.getConnection();
+             PreparedStatement statement =
+                     connection.prepareStatement(SQLConstants.GET_CATEGORIES_BY_ORGANIZATION_SQL)) {
+            statement.setString(1, organization);
+
+            ResultSet rs = statement.executeQuery();
+            while (rs.next()) {
+                String id = rs.getString("UUID");
+                String name = rs.getString("NAME");
+                String description = rs.getString("DESCRIPTION");
+                int tenantID = rs.getInt("TENANT_ID");
```

Comment:
Can we get rid of TENANT_ID from this table?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/10585/commits/37158abec84c566087fdcf7e29c9fd27849a9f6c